src/utils/utils.service.ts

```diff
@@ -4,15 +4,28 @@ import { Coin as TerraCoin, InjectLCDClient, LCDClient } from 'nestjs-terra'
 import { LCDClientError } from 'src/common/errors'
 import { Coin, Validator } from 'src/common/models'
 import { ValidatorVotingPower } from './models'
+import { ConfigService } from '@nestjs/config'
 
 @Injectable()
 export class UtilsService {
+  private nativeLcdService?: LCDClient = undefined
+
   constructor(
     @InjectPinoLogger(UtilsService.name)
     private readonly logger: PinoLogger,
     @InjectLCDClient()
     private readonly lcdService: LCDClient,
-  ) {}
+    private readonly config: ConfigService,
+  ) {
+    const native_lcd_url = this.config.get<string>('NATIVE_LCD_URL')
+    const native_lcd_chain_id = this.config.get<string>('NATIVE_LCD_CHAIN_ID')
+    if (native_lcd_url && native_lcd_chain_id) {
+      this.nativeLcdService = new LCDClient({
+        URL: native_lcd_url,
+        chainID: native_lcd_chain_id,
+      })
+    }
+  }
```

Comment:
support to add an additional LCD url, since mantlemint does not support some LCD queries like blockInfo

Reply:
For the capsulization, we usually route queries which are not supported by mantlemint into rpc node but use same LCD URL, so hive doesn't have to be conserned with that.